spec/std/isa/inst/I/fence.yaml

```diff
@@ -6,14 +6,14 @@
 $schema: "inst_schema.json#"
 kind: instruction
 name: fence
-long_name: Memory oxdering fence
+long_name: Memory ordering fence
```

Comment:
It would be nice to push these sorts of changes via a separate PR, but not a mortal sin.